### PR TITLE
arandr: 0.1.10 -> 0.1.11

### PR DESCRIPTION
**Pull request by borg1622**
###### Description of changes

- [ARandR Changelog](https://github.com/chrysn/arandr/blob/release/ChangeLog)
  - Fix Python 3.11 compatibility by replacing getargspec with getfullargspec

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lheckemann** [comment]
Looks good, but the commits don't follow the convention in https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md and don't seem to be associated with your github account -- maybe check that your git identity email address is associated with your github account :)

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7183,6 +7183,12 @@
       fingerprint = "1412 816B A9FA F62F D051 1975 D3E1 B013 B463 1293";
     }];
   };
+  itcger = {
+    email = "nixpkgs@dirk-osburg.de";
+    github = "itc-ger";
```
Comment (fgaz) [inline maintainers/maintainer-list.nix:7188]: This doesn't match your current username

**Review by fgaz — CHANGES_REQUESTED** [review]
"added itcger to list of maintainers" should be "maintainers: add itcger"

edit: well, also change itcger to your actual username

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (borg1622) [reply maintainers/maintainer-list.nix:7188]: That's right. I know I have to correct these things - will do this soon.

**Comment by domenkozar** [comment]
https://github.com/NixOS/nixpkgs/pull/243463

**Comment by lheckemann** [comment]
Superseded by #263632

**Pull request closed without merging**